pyomo/contrib/solver/solvers/gurobi_direct.py

```diff
@@ -229,9 +220,43 @@ def version(self):
         )
         return version
 
+
+class GurobiDirect(GurobiSolverMixin, SolverBase):
+    """
+    Interface to Gurobi direct (not persistent)
```

Comment:
I would change this to something like "Interface to Gurobi using gurobipy"

Reply:
Why is this one marked resolved?

Reply:
Because I updated the comment!

Reply:
Odd. I must not have refreshed my page or something. Apologies.